src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlString.cs

```diff
@@ -12,40 +13,28 @@ namespace Microsoft.AspNet.Mvc.Rendering
     /// </summary>
     public class HtmlString : IHtmlContent
     {
-        private static readonly HtmlString _empty = new HtmlString(string.Empty);
         private readonly string _input;
 
         /// <summary>
-        /// Instantiates a new instance of <see cref="HtmlString"/>.
+        /// Returns an <see cref="HtmlString"/> with empty content.
         /// </summary>
-        /// <param name="input"><c>string</c>to initialize <see cref="HtmlString"/>.</param>
-        public HtmlString(string input)
-        {
-            _input = input;
-        }
+        public static readonly HtmlString Empty = new HtmlString(string.Empty);
 
         /// <summary>
-        /// Returns an <see cref="HtmlString"/> with empty content.
+        /// Creates a new instance of <see cref="HtmlString"/>.
         /// </summary>
-        public static HtmlString Empty
+        /// <param name="input"><c>string</c>to initialize <see cref="HtmlString"/>.</param>
+        public HtmlString(string input)
         {
-            get
-            {
-                return _empty;
-            }
+            _input = input;
         }
 
-        /// <summary>
-        /// Writes the value in this instance of <see cref="HtmlString"/> to the target
-        /// <paramref name="writer"/>.
-        /// </summary>
-        /// <param name="writer">The <see cref="TextWriter"/> to write contents to.</param>
-        /// <param name="encoder">The <see cref="IHtmlEncoder"/> with which the output must be encoded.</param>
-        public void WriteTo(TextWriter writer, IHtmlEncoder encoder)
+        /// <inheritdoc />
+        public void WriteTo([NotNull] TextWriter writer, [NotNull] IHtmlEncoder encoder)
         {
             writer.Write(_input);
         }
-
+        
```

Comment:
Get thee the trailing whitespace visualizer (which also automagically nukes trailing whitespace).